ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -672,124 +543,28 @@ export default class ConfirmApproveContent extends Component {
-                        },
-                      )
-                    : getAccountLink(
-                        toAddress,
-                        chainId,
-                        {
-                          blockExplorerUrl: rpcPrefs?.blockExplorerUrl ?? null,
-                        },
-                        null,
-                      );
-                  global.platform.openTab({
-                    url: blockExplorerTokenLink,
-                  });
-                }}
-                target="_blank"
-                rel="noopener noreferrer"
-                title={t('etherscanView')}
-              >
-                <i
-                  className="fa fa-share-square fa-sm"
-                  style={{ color: 'var(--color-icon-default)', fontSize: 11 }}
-                  title={t('etherscanView')}
-                />
-              </Button>
-            </Box>
-          </Box>
-        ) : (
-          <Box marginBottom={4} marginTop={2}>
-            <Button
-              type="link"
-              className="confirm-approve-content__verify-contract-details"
-              onClick={() => this.setState({ setshowContractDetails: true })}
-            >
-              {t('verifyContractDetails')}
-            </Button>
-            {setshowContractDetails && (
-              <ContractDetailsModal
-                onClose={() => this.setState({ setshowContractDetails: false })}
-                tokenName={tokenSymbol}
-                tokenAddress={tokenAddress}
-                toAddress={toAddress}
-                chainId={chainId}
-                rpcPrefs={rpcPrefs}
-                tokenId={tokenId}
-                assetName={assetName}
-                assetStandard={assetStandard}
-              />
-            )}
-          </Box>
-        )}
-        {assetStandard === TokenStandard.ERC20 ? (
-          <div className="confirm-approve-content__edit-submission-button-container">
-            <div
-              className="confirm-approve-content__medium-link-text cursor-pointer"
-              onClick={() =>
-                showEditApprovalPermissionModal({
-                  customTokenAmount,
-                  decimals,
-                  origin,
-                  setCustomAmount,
-                  tokenAmount,
-                  tokenSymbol,
-                  tokenBalance,
-                })
-              }
-            >
-              {t('editPermission')}
-            </div>
-          </div>
-        ) : null}
+        <Box marginBottom={4} marginTop={2}>
+          <Button
+            type="link"
+            className="confirm-approve-content__verify-contract-details"
+            onClick={() => this.setState({ setshowContractDetails: true })}
+          >
+            {t('verifyContractDetails')}
+          </Button>
+          {setshowContractDetails && (
+            <ContractDetailsModal
+              onClose={() => this.setState({ setshowContractDetails: false })}
```

Comment:
In a follow up PR we should change `setshowContractDetails` to `setShowContractDetails` but I won't block on that.